configs/unet/fcn_unet_s5-d16_4x4_256x864_160k_kitti.py

```diff
@@ -0,0 +1,13 @@
+_base_ = [
+    '../_base_/models/fcn_unet_s5-d16.py', '../_base_/datasets/kitti_seg.py',
+    '../_base_/default_runtime.py', '../_base_/schedules/schedule_160k.py'
+]
+
+model = dict(
+    decode_head=dict(num_classes=19),
+    auxiliary_head=dict(num_classes=19),
+    # model training and testing settings
+    train_cfg=dict(),
+    test_cfg=dict(mode='whole'))
+
+data = dict(samples_per_gpu=4, workers_per_gpu=4)
```

Comment:
May remove this line, since it is no changes from `../_base_/datasets/kitti_seg.py`.